tiledb/sm/cpp_api/array_schema.h

```diff
@@ -36,6 +36,7 @@
 #define TILEDB_CPP_API_ARRAY_SCHEMA_H
 
 #include "attribute.h"
+#include "dimension_label_experimental.h"
```

Comment:
I think we should avoid adding the experimental C API to the non-experimental C++ API.